integration-test/common/src/test/java/org/apache/uniffle/test/CoordinatorReconfigureNodeMaxTest.java

```diff
@@ -131,9 +131,11 @@ public void testReconfigureNodeMax() throws Exception {
     assertEquals(5, info.getServerToPartitionRanges().keySet().size());
 
     // case3: recover its value to 10
+    Thread.sleep(1000L);
```

Comment:
We should `Awaitility.await`.

Reply:
@maobaolong I added a new comment above this line. This sleep to make sure the last modification time of the `tempConfFilePath` file changed.

Reply:
You wait for a modification timestamp instead `sleep`.